Cargo.toml

```diff
@@ -13,8 +13,13 @@ keywords = ["contour", "polygon", "isoring", "marching-squares", "geojson"]
 license = "MIT/Apache-2.0"
 
 [dependencies]
-geojson = ">=0.16, <=0.24"
+geojson = { version = ">=0.16, <=0.24", optional = true }
+geo-types= { version = "0.7.0" }
 lazy_static = "1.0"
 serde_json = "^1.0"
 rustc-hash = "1.0"
 slab = "0.4"
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
This, in tandem with the `#![cfg_attr(docsrs, feature(doc_auto_cfg))]` in lib.rs, allows us to document feature gated methods like this:
<img width="1044" alt="Screen Shot 2022-09-22 at 12 31 27 PM" src="https://user-images.githubusercontent.com/217057/191841950-040a5cd1-444c-42b9-8f9d-45301c08ceaf.png">

Reply:
Great, thanks !